README.md

```diff
@@ -1,15 +1,31 @@
-# OpenSearch
+<img src="https://github.com/opensearch-project/documentation-website/blob/main/assets/images/logo.svg" height="64px"/>
 
-Hello!
+We believe that open source makes the world a better place. Sadly, [Elastic
+disagrees](https://opensource.org/node/1099).
 
-OpenSearch is an open source search and analytics engine derived from Elasticsearch 7.10.2, and is currently in an alpha state. While we're still in the process of getting set up, we are excited to build the best distributed search engine on the planet with you.
+**OpenSearch** is [a fork and spiritual successor](https://aws.amazon.com/blogs/opensource/introducing-opensearch/)
```

Comment:
It's "a community-driven, open source fork of Elasticsearch and Kibana"

Reply:
The language used in opensearch.org is "OpenSearch is a community-driven, open source search and analytics suite derived from Apache 2.0 licensed Elasticsearch 7.10.2 & Kibana 7.10.2." I would keep the line here aligned to the language on the website.